src/utils/helpers.ts

```diff
@@ -54,6 +63,52 @@ const getPlatformInfo = (platformName: string) => {
   else return { shortName: '', icon: {} };
 };
 
+const createDevtoolsDocsUrl = (
+  supportedFramework: string,
+  urlReadySuite: string,
+  suite: string,
+) => {
+  let linkSupported = true;
+
+  let devtoolsDocsURL = `${baseDocsURL}/devtools/tests/${supportedFramework}.html#${urlReadySuite}`;
+  if (
+    suite === removedOldTestDevTools ||
+    nonDocumentedTestsDevTools.includes(suite)
+  ) {
+    devtoolsDocsURL = '';
+    linkSupported = false;
+  }
+  return { devtoolsDocsURL, linkSupported };
+};
+
+const getDocsURL = (suite: string, framework_id: Framework['id']) => {
+  const framework = frameworkMap[framework_id];
+  const supportedFramework =
+    supportedPerfdocsFrameworks[framework as SupportedPerfdocsFramework];
+  const urlReadySuite = suite.replace(/:|\s|\.|_/g, '-').toLowerCase();
+
+  let docsURL = '';
+  let isLinkSupported = true;
+
+  const isDevToolsFramework = framework_id === devToolsFramework.id;
+
+  if (isDevToolsFramework) {
```

Comment:
`framework_id === devToolsFramework.id;` can be used directly at the if clause. It's easy to read.
```suggestion
  if (framework_id === devToolsFramework.id) {
```